spacepy/irbempy/__init__.py

```diff
@@ -894,7 +850,7 @@ def find_LCDS_K(ticks, K, extMag='T01STORM', options=[1, 1, 3, 0, 0], omnivals=N
                 continue
 
             # print('L* at inner bracket: {0}'.format(LS1['Lstar']))
-            LCDS, LCDS_PA = LS1['Lstar'][0], pa
+            LCDS, LCDS_PA = LS1['Lstar'][0, 0], pa[0]
```

Comment:
This feels inconsistent because pa is treated as a scalar above, in the assignment to results. Does AlphaOfK return a single value or an array of length 1? Should probably document it.

Reply:
Aye it's always an array. This was causing a previously unhit numpy deprecation warning (and somehow an error in the CI... that was a frustrating one until I read more into the deprecation warning).

Reply:
But every place else pa is treated as a scalar: `results['AlphaEq'][idxt, idxk] = pa`. It's going to be `nan` if `GEOcoord[0]` isn't finite (which is a bit weird but I guess the assumption is that all three dimensions will be bad for a bad coordinate).

Do we treat this as "irbempy needs more work and this is out of scope for this PR"?